test/functional/test_framework/messages.py

```diff
@@ -326,22 +321,20 @@ class CBlockLocator:
     __slots__ = ("nVersion", "vHave")
 
     def __init__(self):
-        self.nVersion = MY_VERSION
         self.vHave = []
 
     def deserialize(self, f):
-        self.nVersion = struct.unpack("<i", f.read(4))[0]
+        struct.unpack("<i", f.read(4))[0]  # Ignore version field.
```

Comment:
I think It should be:
`f = struct.unpack("<i", f.read(4))[0]`
instead of 
`struct.unpack("<i", f.read(4))[0]`

because you're not using the return value..

Reply:
`f` is the stream that we're deserializing from. We intentionally discard the version field from this message.